client/src/App.jsx

```diff
@@ -53,11 +53,17 @@ export default function App() {
 
         // BUG: SOMETIMES THERE ARE DUPLICATES!!!!!!!!!
         // Filtered on nationality because then the names get too hard :p
-        const apiParams = "?format=JSON&nat=CA,US&results=" + numOfRandomUsers
+        
```

Comment:
i dont like the extra new lines here. please remove them

Reply:
I should've known I'll fix it